sam/header.go

```diff
@@ -111,6 +111,10 @@ type tagPair struct {
 	value string
 }
 
+func (t tagPair) String() string {
```

Comment:
Please remove this; `fmt.Stringer` gets in the way of debugging prints with `%+v`.